### PR TITLE
Fix variables name for User entity commands

### PR DESCRIPTION
**Pull request by furkanaltaca**

### CODE REVIEW

File: Business/Handlers/Users/Commands/CreateUserCommand.cs [hunk 1]
```diff
@@ -45,9 +45,9 @@ public CreateUserCommandHandler(IUserRepository userRepository)
             [LogAspect(typeof(FileLogger))]
             public async Task<IResult> Handle(CreateUserCommand request, CancellationToken cancellationToken)
             {
-                var userExits = await _userRepository.GetAsync(u => u.Email == request.Email);
+                var userExists = await _userRepository.GetAsync(u => u.Email == request.Email);
```
Comment (namikbarisidil) [inline Business/Handlers/Users/Commands/CreateUserCommand.cs:48]: I think naming itself is wrong here. "[something]Exists" referring to a boolean value where this is an object. I think it should be "user". Valid for other changes.

**Review by namikbarisidil — CHANGES_REQUESTED** [review]

File: Business/Handlers/Users/Commands/CreateUserCommand.cs (hunk 1 above)
Comment (furkanaltaca) [reply Business/Handlers/Users/Commands/CreateUserCommand.cs:48]: It was just a word mistake. I wanted to fix it when I realized it.

Comment (namikbarisidil) [reply Business/Handlers/Users/Commands/CreateUserCommand.cs:48]: I know. While correcting the typo, it is also a good practice to correct the terminology.

**Comment by keremvaris** [comment]
thank you for the information

**Pull request closed without merging**